ui/imports/shared/views/AssetsViewAdaptor.qml

```diff
@@ -147,7 +147,7 @@ QObject {
 
                 filters: [
                     FastExpressionFilter {
-                        expression: root.chains.includes(model.chainId)
+                        expression: root.chains.includes(model.chainId.toString())
```

Comment:
Why is it needed? Looks a bit suspicious like sth that may lead to bugs in the future :thinking:

Reply:
I think the chainIds coming from nim models are always ints. This one is for sure. I'll update the model expected roles as well

Reply:
I could instead remove the toString here and make sure the `root.chains` is an array of numbers

Reply:
Ideally we should have the same type for all `key`/`id` roles from the backend. But probably not possible to adjust it now :sweat: